news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md

```diff
@@ -0,0 +1,183 @@
+* [#103](https://github.com/PrestaShop/ps_facetedsearch/pull/103): Fix facet display when there is only one filter, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#104](https://github.com/PrestaShop/ps_facetedsearch/pull/104): Split currency if a ; is present in the format string, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#105](https://github.com/PrestaShop/ps_facetedsearch/pull/105): Fix for cache problem when used as widget. Thank you [@iqit-commerce](https://github.com/iqit-commerce)
+* [#106](https://github.com/PrestaShop/ps_facetedsearch/pull/106): Bump to 3.0.6, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#107](https://github.com/PrestaShop/ps_facetedsearch/pull/107): Release 3.0.6, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#109](https://github.com/PrestaShop/ps_facetedsearch/pull/109): Fix ps_facetedsearch.tpl vars. Thank you [@davidglezz](https://github.com/davidglezz)
+* [#110](https://github.com/PrestaShop/ps_facetedsearch/pull/110): Do not use seoUrl if user do not want to use it, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#111](https://github.com/PrestaShop/ps_facetedsearch/pull/111): Fix ie11 compatibility, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#112](https://github.com/PrestaShop/ps_facetedsearch/pull/112): Bump to 3.1.0, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#113](https://github.com/PrestaShop/ps_facetedsearch/pull/113): Release 3.1.0, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+### Google Adwords
+
+* [#6](https://github.com/PrestaShop/gadwords/pull/6): Deploy v2.0.0 of the gadwords module, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#7](https://github.com/PrestaShop/gadwords/pull/7): Fix typo error on module installation, by [@apacios](https://github.com/apacios)
+* [#8](https://github.com/PrestaShop/gadwords/pull/8): Bump version to 2.0.1, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#9](https://github.com/PrestaShop/gadwords/pull/9): Deploy v2.0.1 of the gadwords module, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+### Auto Upgrade
+
+* [#284](https://github.com/PrestaShop/autoupgrade/pull/284): Display a notice if PHP is below v7.1, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#325](https://github.com/PrestaShop/autoupgrade/pull/325): Prepend and optimize autoloader, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#326](https://github.com/PrestaShop/autoupgrade/pull/326): Add PHPDoc & log when module switch the current theme, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#327](https://github.com/PrestaShop/autoupgrade/pull/327): Update logs around zip upgrade, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#328](https://github.com/PrestaShop/autoupgrade/pull/328): Bump version to 4.9.0, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+### classic-rocket
```

Comment:
Do we list classic-rocket notifs now ?

Reply:
We usually list all public repositories on the org.
classic-rocket is in the org.
maybe it's time to discuss with the author and see if he agrees to move to "friends of presta" org

Reply:
Why not ask the author first then?

Reply:
the target was to add classic-rocket to the project, but that seems to become a difficult option... 
now that friend of presta exists, it should become the place for incubation.

Reply:
Let's discuss the place of "Friends of Presta" organization next week with @toutantic.

I'm totally open to "welcome" this theme into the organization, and **I'd like to be allowed to share the community resources** into the official documentation.

We have so many useful modules that could be used as documentation...

Reply:
I agree. useful resources from the community should have a place in the devdoc.